modules/rostests/winetests/ddraw/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+
+add_definitions(-DUSE_WINE_TODOS)
+add_executable(ddraw_winetest d3d.c ddraw1.c ddraw2.c ddraw4.c ddraw7.c ddrawmodes.c dsurface.c refcount.c visual.c testlist.c)
```

Comment:
Use a 'SOURCE' list?

Reply:
yes and with vertical layout, which improves git-blame-ability when the list is extended.